Makefile

```diff
@@ -8,8 +8,12 @@ VERSION ?= 2.5.0-beta.6
 # MIN_KUBE_VERSION is the build flag of minimum Kubernetes version.
 MIN_KUBE_VERSION ?= 1.18.0
 
-# Generate kuttl e2e tests for the following storageos/kind-node versions.
+# Generate kuttl e2e tests for the following storageos/kind-node versions
+# # TEST_KIND_NODES is not intended to be updated manually.
+# # Please edit LATEST_KIND_NODE instead and run 'make update-kind-nodes'.
 TEST_KIND_NODES ?= 1.18.6,1.19.0,1.20.5,1.21.0,1.22.3
+
+LATEST_KIND_NODE ?= 1.22.3
```

Comment:
This piece of info looks a temporary thing to be able to update `TEST_KIND_NODES`. If i'm right i suggest to remove the line here and check variable at `update-kind-nodes`, like
```
update-kind-nodes:
ifndef LATEST_KIND_NODE
	$(error LATEST_KIND_NODE is undefined)
endif
LATEST_KIND_NODE=$(LATEST_KIND_NODE) ./hack/update-kind-nodes.sh
```

On this way we enforce the variable but didn't persist in repo.